ocs_ci/utility/rhv.py

```diff
@@ -409,3 +410,48 @@ def remove_disks(self, vm, detach_only=True):
                 self.remove_disk(
                     vm=vm, identifier=disk.id, key="id", detach_only=detach_only
                 )
+
+    def get_vm_status(self, vm_name):
+        """
+        Get the power status of RHV VM
+
+        Args:
+           vm_name (str): RHV VM name
+
+        Returns :
+           str: Power status of RHV VM
+
+        """
+        vm = self.get_vms_by_pattern(pattern=vm_name)[0]
+        vm_service = self.vms_service.vm_service(vm.id)
+        vm_info = vm_service.get()
+        return vm_info.status
+
+    def stop_rhv_vms(self, vm_names, force=False):
+        """
+         Shutdown the RHV virtual machines
+
+        Args:
+            vm_names (list): Names of RHV vms
+            force (bool): True for non-graceful VM shutdown, False for
+                graceful VM shutdown.
+
+        """
+        for vm_name in vm_names:
+            # Find the virtual machine
+            vm = self.get_vms_by_pattern(pattern=vm_name)[0]
+            vm_service = self.vms_service.vm_service(vm.id)
+            vm_service.stop(force=force)
+            # Wait till the virtual machine is down:
+            try:
+                for status in TimeoutSampler(600, 5, self.get_vm_status, vm_name):
```

Comment:
Maybe it's a better idea to receive the 600 as a parameter here? or at least a parameter with 600 as default? 
Like you did with `def stop_nodes(self, nodes, timeout=600, wait=True, force=True):` Not sure how much time the stopping node task should take but anyway I just thought it could be nice here :)

Reply:
Done